src/utils/filterContests.test.js

```diff
@@ -0,0 +1,18 @@
+import { filterContests } from './filterContests';
+
+function testFilter(contests, from, to) {
+  let filteredContests = contests.filter((contest) => {
+    return contest.startTime >= from && contest.startTime <= to;
+  });
+  contests = filterContests(contests, from, to);
+  return expect(contests).toMatchObject(filteredContests);
+}
+
+test('filter contests', () => {
+  let cs = [];
+  for(let i = 1; i <= 10; i++)
+    cs.push({
+      startTime: new Date(i * 1000)
+    });
+  return testFilter(cs, new Date(1000), new Date(7000));
```

Comment:
Frankly, I don't think this test is doing much. Nobody is going to filter contests with timestamps like these, so I'd say to use something like real Dates like "March 27, 2018".

Reply:
Using the object mother pattern, we can actually use stuff like `mother.today()` or `mother.yesterday()` also